[liqoctl] set gateway server Service location in `peer` command (#2913)
liqotech/liqo: docs/usage/peer.md

The peering command enables all 3 liqo modules and performs the following steps:

1. **enables networking**.

> Exchanges network configurations and creates the two **gateways** (one server in a cluster and one client in the other) to let the two clusters communicate over a secure tunnel.

Reviewer: ```suggestion
Exchanges network configurations and creates the two **gateways** (one acting as _server_ and located in the provider cluster, another acting as _client_ in the consumer cluster) to let the two clusters communicate over a secure tunnel.
```

> By default the gateway server in placed in the provider, while the gateway client in the consumer, but you can configure it with the `--server-service-location` flag.

Reviewer: ```suggestion
By default the gateway server is located in the provider, while the gateway client in the consumer; this can be customized with the `--server-service-location` flag in `liqoctl`.
```
